src/main/scala/TextToStructuredData.scala

```diff
@@ -0,0 +1,25 @@
+import org.apache.spark.sql.functions._
+import TweetsNormalization.normalizeTweets
```

Comment:
I see that there is no package structure defined for the artifacts. This is not how Java/Scala software developed nowadays. 
Just compare the line to the previous one and you'll see what I'm taking about:
```scala
org.apache.spark.sql.functions._
```
Please learn about how to structure packages and what should be naming for open source and commercial apps. Here is just an example where you can find the info: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
here is related info as well: http://docs.scala-lang.org/style/naming-conventions.html#packages